packages/storybook/elements-stencil-docs.json

```diff
@@ -5411,6 +5411,25 @@
           "optional": false,
           "required": false
         },
+        {
+          "name": "delay",
+          "type": "[number, number] | number",
```

Comment:
Im Code ist die Reihenfolge `number -> [number, number]`. Das macht mMn auch am meisten Sinn, weil man wahrscheinlich meistens nur einen Wert für den Delay angibt statt `show` und `hide` separat zu setzen.
Und man könnte noch einen default Wert setzen (zum Beispiel 0).

```
"type": "number | [number, number | null] | null]",
"default": "0",
```

Reply:
This is auto-generated so we cannot do anything about the order of the typing. But I would also add a default value to the property (see above).